README.md

```diff
@@ -1 +1,50 @@
-# ramen
+# Ramen
+
+Ramen is a **disaster-recovery orchestrator** for stateful applications across
+a set of peer kubernetes clusters that,
```

Comment:
I believe a colon is usually preferred over a comma to begin a list: https://www.grammarly.com/blog/bullet-points/#:~:text=If%20the%20text%20introducing%20your,numbered%20list%20rather%20than%20bullets.

Appears in multiple places below and in install.md.